Doc/library/faulthandler.rst

```diff
@@ -179,3 +181,10 @@ handler:
       File "<stdin>", line 1 in <module>
     Segmentation fault
 
+.. seealso::
+
+   Module :mod:`pdb`
+      Interactive source code debugger for Python programs.
+
+   Module :mod:`traceback`
+     Standard interface to extract, format and print stack traces of Python programs.
```

Comment:
```suggestion
      Standard interface to extract, format and print stack traces of Python programs.
```

Minor nit, but no reason not to be consistent with spacing, both viz. the above line and the general Python docs convention here.